### DISCUSSION TITLE
When I looked up this library I expected the table model to be different.

### DISCUSSION THREAD

**Discussion by AdelinGhanaem** [spatie/laravel-translation-loader#159] [Ideas]
Instead of this Model/Table
```
LanguageLine::create([
   'group' => 'validation',
   'key' => 'required',
   'text' => ['en' => 'This is a required field', 'nl' => 'Dit is een verplicht veld'],
]);
```
Isn't it more convenient to be something like 

LanguageLine::create([
   'group' => 'validation',
   'key' => 'required',
   'en' => 'This is a required field',
   'ar' => 'text in arabic', 
   'gr' => 'text in dutch',  
]);

**Comment by safarovitch** [comment]
I believe in terms of reading and wiring data to database your suggestion is true. But also sql has array notation acces within json or text fields. And also Laravel's $casts property makes it easier to access specific indexed member of array inside text or json field. 

Also since no application has definite number of languages it is easier to maintain json field an make all dynamic rather than creating field for each language each time content gets updated with new translation.
